src/pages/_app.tsx

```diff
@@ -50,6 +50,7 @@ function MyApp({ Component, pageProps }): JSX.Element {
   return (
     <>
       <Head>
+        <link rel="apple-touch-icon" sizes="192x192" href="/images/logo/Logo@192x192.png" />
```

Comment:
this probably shouldn't be here, we have somewhere else where we specify all the meta tags and links